exercises/all-your-base/all_your_base_test.py

```diff
@@ -82,9 +82,10 @@ def test_both_bases_are_negative(self):
     # Utility functions
     def setUp(self):
         try:
-            self.assertRaisesRegex = self.assertRaisesRegexp
+            with self.assertRaisesRegex(AttributeError, r".+"):
+                raise AttributeError('x')
```

Comment:
A cleaner solution is to just call `self.assertRaisesRegex` without actually using it.

```python
try:
    self.assertRaisesRegex
except AttributeError:
    self.assertRaisesRegex = self.assertRaisesRegexp
```